packages/core/src/bags/client/__tests__/getBagOperation.test.js

```diff
@@ -0,0 +1,57 @@
+import { getBagOperation } from '..';
+import {
+  mockBagId,
+  mockBagOperation,
+  mockBagOperationId,
+} from 'tests/__fixtures__/bags';
+import client from '../../../helpers/client';
+import fixtures from '../__fixtures__/getBagOperation.fixtures';
+import moxios from 'moxios';
+
+describe('getBagOperation', () => {
+  const expectedConfig = undefined;
+  const spy = jest.spyOn(client, 'get');
+
+  beforeEach(() => {
+    moxios.install(client);
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => moxios.uninstall(client));
+
+  it('should handle a client request successfully', async () => {
+    const response = mockBagOperation;
+
+    fixtures.success({
+      bagId: mockBagId,
+      bagOperationId: mockBagOperationId,
+      response,
+    });
+
+    expect.assertions(2);
```

Comment:
This instruction is unnecessary since the call to `getBagOperation` is being awaited.

Reply:
Removed in all tests.